### ISSUE TITLE
Question about CD?

### ISSUE THREAD

**Issue by brian3021** [nywang16/Pixel2Mesh#93]
Hi, thanks for sharing the repo.

I have a question about chamfer loss in `losses.py` file.

why you multiply `0.55` to `tf.reduce_mean(dist2)` ?
dist2 means that the **pred** to **gt_pt** right?

and why multiply `3000` to all chamfer loss? Thx.
